react/.eslintrc.json

```diff
@@ -1,32 +1,32 @@
 {
-    "env": {
-        "browser": true,
-        "es2021": true,
-        "jest": true
-    },
-    "extends": [
-        "eslint:recommended",
-        "plugin:react/recommended",
-        "plugin:jsx-a11y/recommended",
-        "plugin:@typescript-eslint/recommended",
-        "prettier/",
-        "prettier/prettier"
-    ],
-    "overrides": [],
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "plugins": ["react", "react-hooks", "@typescript-eslint", "prettier"],
-    "rules": {
-	"prettier/prettier": ["error"],
-        "@typescript-eslint/no-unused-vars": "error",
-        "@typescript-eslint/no-explicit-any": "off"
-    },
-    "settings": {
-        "react": {
-            "version": "detect"
-        }
-    }
+  "env": {
```

Comment:
this just has just shifts things 2 spaces to left.  so first level has 2-spaces indention and then subsequent levels have 4 spaces indention. 

 i think its best we avoid changing linting/indentions on top level things to make reviews easier until we do https://tacc-main.atlassian.net/browse/WG-248.